docs/data-ai/ai/create-dataset.md

```diff
@@ -14,244 +14,169 @@ aliases:
-   ```
+This page explains how to create a dataset that meets these criteria for your training purposes.
+
+## Prerequisites
+
+{{% expand "A machine connected to the Viam app" %}}
 
-1. Continue to [Train a tflite machine learning model](/data-ai/ai/train-tflite/).
+{{% snippet "setup.md" %}}
+
+{{% /expand %}}
+
+{{% expand "A camera, connected to your machine, to capture images" %}}
+
+Follow the guide to configure a [webcam](/operate/reference/components/camera/webcam/) or another [camera component](/operate/reference/components/camera/), if you haven't already.
 
 {{% /expand%}}
 
-## Prerequisites
+## Capture images
+
+{{< tabs >}}
+{{% tab name="Capture individual images directly into a dataset" %}}
+
+You can add images to a dataset directly from a camera or vision component feed in the **CONTROL** or **CONFIGURATION** tabs of the Viam app.
+This technique can help you create a dataset without filling the **DATA** tab with a large number of unhelpful empty images.
+
+To add an image directly from a dataset from a visual feed, complete the following steps:
+
+1. Open the **TEST** panel of any camera or vision component to view a feed of images from the camera.
+1. Click the button marked with the camera icon to save the currently displayed image to a dataset:
+   {{< imgproc src="/components/camera/add_image_to_dataset_button.png" alt="A button marked with the outline of a camera, emphasized in red" resize="800x" style="width:500px" class="imgzoom" >}}
+1. Select an existing dataset.
+1. Click **Add** to add the image to the selected dataset.
+1. When you see a success toast that reads "Saved image to dataset", you have successfully added the image to the dataset.
+
+To view images added to your dataset, go to the **DATA** page's [**DATASETS** tab](https://app.viam.com/data/datasets) in the Viam app. Select the dataset to view the images within that dataset.
 
-{{< expand "At least 10 captured images of what you want your machine to recognize." >}}
+{{< alert title="Tip" color="tip" >}}
 
-[Capture and sync image data](/data-ai/capture-data/capture-sync/) using the data management service.
+Use this technique to manually add tricky images where your first model version fails to your training dataset.
+Annotate the tricky images, then train a new version of your model to improve your model.
 
-When training machine learning models, it is important to supply a variety of images.
-The dataset you create should represent the possible range of visual input.
-This may include capturing images of different angles, different configurations of objects and different lighting conditions.
-The more varied the provided dataset, the more accurate the resulting model becomes.
+{{< /alert >}}
 
-{{< /expand >}}
+{{% /tab %}}
+{{% tab name="Capture bulk images" %}}
+
+To capture a large number of images for training your ML models, [Capture and sync image data](/data-ai/capture-data/capture-sync/) using the data management service with your camera.
+
```

Comment:
You'd still need to add them to the dataset I think?

Reply:
Ah I see. You add them as the last step. That possibly needs some caveat then because if you followed one method they'll already be in the dataset

Reply:
Noted! I'll reword this for better parity between the tabs.